packages/store-ui/src/SearchSuggestions/Desktop.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import type { FC } from 'react'
+
+import SearchSuggestionsAutocomplete from './Autocomplete'
+import { useSearchSuggestions } from './base/useSearchSuggestions'
+import SearchSuggestionsProduct from './Products'
+import SearchSuggestionsHistory from './SearchHistory'
+import SearchSuggestionsTopSearches from './TopSearches'
+import type { SummaryComponent } from './Products'
+
+interface Item {
+  key: string
+  term: string
+}
+
+interface Props {
+  topSearches: Item[] | undefined
+  history: Item[] | undefined
+  autocomplete: Item[] | undefined
```

Comment:
Why do you need to be defined `undefined`?

Reply:
because I want to force typescript to complain if any argument is not passed by the developer